src/sagas/pushNotification.js

```diff
@@ -246,10 +177,21 @@ export function* onAppInitialization() {
     return;
   }
 
-  yield call(installForegroundListener);
+  const deviceId = yield call(getDeviceId);
+  // const deviceId = yield call(getDeviceId);
+  if (!deviceId) {
+    console.debug('Halting push notification initialization because the device id is null.');
+    yield put(onExceptionCaptured(new Error('Device id is null')));
+    return;
+  }
+
+  const isForegroundListernerInstalled = yield call(installForegroundListener);
```

Comment:
Is this a typo? Listerner? In that case, you would be creating a variable that already exists (`isForegroundListenerInstalled `), a new name should be used

Reply:
[29e6c64](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/233/commits/29e6c64bb75a8b011986d14176eba98b59fd5b35)